src/components/RelatedCard/RelatedCard.component.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import { Link } from 'react-router-dom';
+import { useStyles } from './styles';
+
+export default function RelatedCard({videoInfo}) {
+
+    console.log(videoInfo);
+
+    const classes = useStyles();
```

Comment:
you are not passing a theme object, so I think that's the reason you are missing more than several styles at your components.  Do not want to try styled-components instead?